package.json

```diff
@@ -43,7 +43,7 @@
     "@types/express": "^4.17.3",
     "@types/jest": "^25.1.4",
     "@types/morgan": "^1.9.0",
-    "@types/node": "^13.9.3",
+    "@types/node": "^12.12.38",
```

Comment:
I thought we were stuck back at node v10?

Reply:
Not for this project:
https://github.com/gnosis/dex-price-estimator/blob/master/.travis.yml#L4

The current LTS release is `12.16.3`